### ISSUE TITLE
Failure when trying to use multi-line text

### ISSUE THREAD

**Issue by isaacrj** [inlife/nexrender#266]
Nexrender fails when using a mutli-line string as value for an asset, for example, trying to use:
"Line 1
Line 2"
as a value for a text layer.

In json, this would be coded as an asset like this:
```json
{
   "type":"data",
    "composition":"*",
    "layerName":"text-layer",
    "property":"Source Text",
    "value":"Line 1\n Line2"
}
```

Nexrender then generates the jsx file for the job, but the relevant code for this property renders as:
```javascript
nexrender.selectLayersByName('*', '_txt-descripcion', function(layer) {
var parts = ["Source Text"];
var value = { "value": Line1 
Line2' }
        nexrender.changeValueForKeypath(layer, parts, value);
        return true;
    });
});
```

The new line between Line1 and Line2, makes the jsx file syntactically incorrect, and the job fails. 

I have no experience programming in node, but I suspect that this could be prevented by checking the type of "value" in the method wrapData inside script.js and, if it is a string, replace each "\n" with "\\\\n".

**Comment by inlife** [comment]
That is an interesting problem to tackle. I don't yet have ideas about how can we solve it. In case anybody has any propositions - please do suggest. :)

**Comment by isaacrj** [comment]
I think that it should be enough to add a type check here:
https://github.com/inlife/nexrender/blob/dace23c8e5ccf36e8a0983f662cae63fe692db2e/packages/nexrender-core/src/tasks/script.js#L37

If `value` is of type `string` then, any occurrence of the new line character `\n` should be replaced with a escaped new line like this: `\\n` . With this replacement, the jsx code related to this issue would read:
```javascript
nexrender.selectLayersByName('*', 'text-layer', function(layer) {
var parts = ["Source Text"];
var value = { "value": Line1\nLine2' }
        nexrender.changeValueForKeypath(layer, parts, value);
        return true;
    });
});
```
 and the problem should be solved.

Unfortunately, I have no experience at all with Node and I don't know how to test this myself.

**Comment by inlife** [comment]
Oh okay, that makes sense. Going to push a fix for that.

**Comment by inlife** [comment]
Published as `v1.8.11`
